# 剩余子弹数量

# 红方1号
int32 red1
# 红方2号
int32 red2
# 蓝方1号
int32 blue1
# 蓝方2号
int32 blue2